frontend/src/metabase/visualizations/echarts/tooltip/index.tsx

```diff
@@ -85,7 +92,13 @@ export const getTooltipBaseOption = (
           "calc(var(--mb-overlay-z-index) + 1)",
         );
 
-        document.body.append(container);
+        if (!isEmbeddingSdk) {
```

Comment:
I don't like inverted conditions, but the main app is the primary case to check here, so i didi it like this.